.gitignore

```diff
@@ -99,9 +99,12 @@ typings/
 # Optional npm cache directory
 .npm
 
-# Optional eslint cache
+# Optional ESLint cache
 .eslintcache
 
+# Optional ESLint configuration
+.eslintrc.json
```

Comment:
I think we'll eventually want to have a canonical eslint configuration for the project too

Reply:
we can discuss it later, in another pull request. I will remove it now.